gradle/libs.versions.toml

```diff
@@ -10,15 +10,18 @@ hilt-android = "2.50"
 hilt-android-compiler = "2.50"
 hilt-compiler = "1.2.0"
 junit = "4.13.2"
+kotlinx-coroutines-test = "1.6.2"
 lifecycle-runtime-ktx = "2.7.0" #Next versions are not stable
 compose-bom = "2024.05.00"
 material-icons-extended = "1.6.7"
+mockk = "1.10.5"
 retrofit = "2.9.0"
 kotlin = "1.9.0"
 kps = "1.9.0-1.0.13"
 dagger = "2.50"
 splash = "1.1.0-rc01"
 truth = "1.1"
+turbine = "0.7.0"
```

Comment:
Android Studio doesn't highlight this for some reason, but there's already `1.1.0` available with lots of new functionality (had the same issue recently)